CNI/romana

```diff
@@ -184,6 +187,52 @@ set_up_pod () {
 
 	# Kubernetes expects valid json on output so no one above should echo anything in stdout
 	echo -n $R
+
+	update_policy "$POD" "$NAMESPACE"
+}
+
+# update_policy discovers romana network policies that affect current pod
+# and notifies romana agent to include new pod into policy
+update_policy () {
```

Comment:
Ah, now we are getting somewhere! :-)

I think that commonly in this context "policy" refers to the policy definition, not the policy implementation.

I think the function could be called "update_policy_implementation" or "add_endpoint_to_policy" or "apply_policy_to_interface" or some such thing.

"update_policy" too strongly implies that the policy definition itself is changed.